packages/manager/apps/web/client/app/private-database/dashboard/translations/Messages_fr_FR.json

```diff
@@ -76,13 +76,14 @@
   "privateDatabase_configuration_event_scheduler": "Statut de l'orchestrateur d'évènements",
   "privateDatabase_configuration_interactive_timeout": "Le nombre de secondes que le serveur attend avant de fermer la connexion lorsqu'il n'y a pas d'activité lors d'une connexion en invite de commande.",
   "privateDatabase_configuration_wait_timeout": "Le nombre de secondes que le serveur attend avant de fermer la connexion lorsqu'il n'y a pas d'activité lors d'une connexion externe.",
-  "privateDatabase_configuration_field_autocommit": "Définit si les transactions SLQ sont automatiquement soumises ou non.",
+  "privateDatabase_configuration_field_autocommit": "Définit si les transactions SQL sont automatiquement soumises ou non.",
   "privateDatabase_configuration_field_event_scheduler": "Permet de déclencher l’exécution de requêtes programmées directement dans le serveur MySQL.",
   "privateDatabase_configuration_field_innodb_buffer_pool_size": "Choix de la taille de la mémoire tampon (en mégaoctets).",
   "privateDatabase_configuration_field_interactive_timeout": "Nombre de secondes pendant lesquelles le serveur attend une activité sur une connexion interactive avant de la fermer.",
   "privateDatabase_configuration_field_max_allowed_packet": "Taille maximum des paquets (en mégaoctets).",
   "privateDatabase_configuration_field_max_connections": "Nombre de connexions simultanées autorisées sur le SQL Privé.",
   "privateDatabase_configuration_field_max_user_connections": "Nombre de connexions simultanées autorisées par utilisateur.",
+  "privateDatabase_configuration_field_sql_mode": "L'option «sql_mode» affecte la syntaxe SQL prise en charge, et les vérifications de validation des données effectués par MySQL/MariaDB. Plus de détails ici : <Lien>",
```

Comment:
**question:** Why using `<Lien>`? Missing URL, isn't it?

Reply:
Hi,

Link will come soon, I choose to proceed that way to have everything production ready for when we will prod our new option (You're right  the translation is not used yet, but will be in the days to come).

I'll remove the unused <link> related sentence, and push this when it will be ready.